avocado/plugins/run.py

```diff
@@ -152,8 +152,11 @@ def configure(self, parser):
                              help="Inject [path:]key:node values into the "
                              "final multiplex tree.")
 
-    def _activate(self, args):
-        # Extend default multiplex tree of --mux_inject values
+    @staticmethod
+    def _process_mux_inject(args):
```

Comment:
Right, my question is: if does it makes sense to combine those two occurrences, that is, create a shared utility function instead of having those two similar functions?